recipe/meta.yaml

```diff
@@ -41,7 +42,6 @@ requirements:
     - expat
     - fontconfig
     - freetype
-    - krb5
```

Comment:
Why was this dropped?

Reply:
It was giving me version conflicts through `qt-main`, it should still be installed by that package.

Reply:
Think re-rendering with both dependencies added to `host` will fix that